test/python/synthesis/test_permutation_synthesis.py

```diff
@@ -24,6 +24,7 @@
     synth_permutation_depth_lnn_kms,
     synth_permutation_basic,
     synth_permutation_reverse_lnn_kms,
+    synth_permutation_depth_lnn_kms,
```

Comment:
I think `synth_permutation_depth_lnn_kms` now appears twice in this import list.